dipy/workflows/tracking.py

```diff
@@ -32,11 +71,10 @@ def _core_run(self, stopping_path, stopping_thr, seeding_path,
         direction_getter = pam
 
         if use_sh:
```

Comment:
There is a problem here. If the use set `--sh_strategy prob` without `--use_sh" the output is deterministic using the pam.

My suggestion is to remove `--use_sh` and rename `--sh_strategy` to `--tracking_method` with the following opton:
- `EuDX`  -> `direction_getter = pam`
- 'deterministic' -> `direction_getter = DeterministicMaximumDirectionGetter.from_shcoeff(pam.shm_coeff, ...)`
- `closetpeaks` -> direction_getter = `ClosestPeakDirectionGetter.from_shcoeff(pam.shm_coeff, ...)`
-`probabilistic` -> direction_getter = `ProbabilisticDirectionGetter.from_shcoeff(pam.shm_coeff, ...)`